tests/console/zypper_lr_validate.pm

```diff
@@ -28,53 +28,35 @@ sub validatelr {
     my $product         = $args->{product};
     my $product_channel = $args->{product_channel} || "";
     my $version         = $args->{version};
```

Comment:
ehm... Shouldn't it be `//` instead of `||` ?

Also, these function calls look fishy to me.

```
 elsif (check_var('FLAVOR', 'Server-DVD-HA')) {
            validatelr({product => "SLES", uri => $uri, version => $version});
            validatelr({product => 'SLE-*HA', uri => get_var('ADDONURL_HA') || "$dvd:///", version => $version});
            if (exists $h_addonurl{geo} || exists $h_addons{geo}) {
                validatelr({product => 'SLE-*HAGEO', uri => get_var('ADDONURL_GEO') || "$dvd:///", version => $version});
            }
            delete @h_addonurl{qw(ha geo)};
            delete @h_addons{qw(ha geo)};
        }
```

Reply:
I just did necessary changes, whole zypper_lr_validate should be reviewer/refactored.

Reply:
Agreed, just create a followup ticket to not let the idea vanish.

Reply:
https://progress.opensuse.org/issues/66451